test/e2e/storage/vsphere/vsphere_zone_support.go

```diff
@@ -518,7 +518,7 @@ func verifyPVZoneLabels(client clientset.Interface, timeouts *framework.TimeoutC
 	ginkgo.By("Verify zone information is present in the volume labels")
 	for _, pv := range persistentvolumes {
 		// Multiple zones are separated with "__"
-		pvZoneLabels := strings.Split(pv.ObjectMeta.Labels["failure-domain.beta.kubernetes.io/zone"], "__")
+		pvZoneLabels := strings.Split(pv.ObjectMeta.Labels[v1.LabelTopologyZone], "__")
```

Comment:
For this, I am wondering if we should also keep it backwards... But I guess maybe it is okay. We just need to make sure we dont run newer version e2e test against old version? @xing-yang I will let you decide.

Reply:
to run older version of tests against older k8s release, we can checkout older k8s release branch or tag and run e2e tests from there.